libcodechecker/server/api/store_handler.py

```diff
@@ -103,11 +103,21 @@ def collect_paths_events(report, file_ids, files):
 
     for event in events:
         file_path = files[event['location']['file']]
+
+        start_loc = event['location']
+        end_loc = event['location']
+        # Range can provide more precise location information
+        # use that if available.
```

Comment:
I think this is a new sentence so this should be started with uppercase: `Use that if available`.

Reply:
I think our current schema clearly not reflect the plists. Multiple ranges are not supported. I think this deserves at least a fixme comment.